rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html

```diff
@@ -0,0 +1,131 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags wagtailsettings_tags static %}
+{% get_settings %}
+
+{% block body_class %}
+    app--editorial-event-landing
+{% endblock %}
+
+{% block content %}
+
+    <div class="page page--no-hero">
+
+        <header class="page__header bg bg--dark">
+            <div class="title-area title-area--landing-page grid">
+                <div class="title-area__content">
+                    {% if page.title %}
+                        <h1 class="title-area__heading heading heading--display-two">{{ page.title }}</h1>
+                    {% endif %}
+                </div>
+            </div>
+
+        </header>
+
+        <div class="page__content">
+
+            {% include "patterns/molecules/jump-nav/jump-nav.html" with modifier="bg bg--dark" sticky=True item=tabs %}
+
+            {# News #}
+            <section class="section section--start bg bg--dark anchor-heading" id="news">
+                <div class="section__row section__row--last schools__about--news">
+                    <div class="section__header grid">
+                        <h2 class="section__heading section__heading--primary heading heading--two">News</h2>
+                    </div>
+                    <div class="section__content grid">
+                        {% include "patterns/organisms/news/news_api_content.html" with news=page.news news_view_all=page.news_view_all modifier="news--single-feature" %}
+                    </div>
+                </div>
+            </section>
+
+            {# Events #}
+            <section class="section section--start anchor-heading bg bg--dark" id="events">
+                <div class="section__row section__row--last schools__about--news">
```

Comment:
should `schools__about--news` be `schools__about--events`?

Reply:
ah just seen a few other news related bits in this section (`modifier="news--single-feature"` on line 47) so perhaps they're all being shared

Reply:
we can ditch the school classes, copied from the schools template and forgot to remove, good spot :)